---
title: InvalidBaseImagePlatform
description: Base image platform does not match expected target platform
---

## Output

```text
```

## Description

## Examples

❌ Bad:

```dockerfile
```

✅ Good:

```dockerfile
```

